mocha/exports.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+if (!process.env.TRAVIS) require('../test/_fake-ci')()
```

Comment:
@Guria Thanks for helping out on the repo :smile: We've added you as a collaborator! :tada: